test_xsd.pl

```diff
@@ -60,28 +60,28 @@
 % Term --> String
 
 test(time_string, S == "2016-01-30") :-
-    xsd_time_string(date(2016,01,30), Y, S),
+    xsd_time_string(date(2016,1,30), Y, S),
```

Comment:
Why this change?  I wrote 01 instead of 1 for the visual compatibility with normal human ways to write down a date.  For Prolog `1` and `01` are the same.

Reply:
Yes you're right, that's just me being pedantic.  I've changed this back and made the use of a leading zero consistent throughout the test file.